collectors/slabinfo.plugin/metadata.yaml

```diff
@@ -43,26 +43,26 @@ overview:
         Due to the large number of metrics generated by this integration, it is disabled by default and must be
```

Comment:
We state this message, but I would then expect a prerequisite item or something telling me how to enable it. :thinking:

Reply:
Last commit addresses the request.